### ISSUE TITLE
Bug: receiving error openvpn: no server found for regions united kingdom and group "premium tcp europe"

### ISSUE THREAD

**Issue by adamsdavis1976** [qdm12/gluetun#275]
**TLDR**: *Describe your issue in a one liner here*

1. Is this urgent?

    - [x] Yes
    - [] No

2. What VPN service provider are you using?

    - [ ] PIA
    - [ ] Mullvad
    - [ ] Windscribe
    - [ ] Surfshark
    - [x] Cyberghost

3. What's the version of the program?

   Running version latest built on 2020-10-27T03:29:10Z (commit edc08c4)

4. What are you using to run the container?

    - [ ] Docker run
    - [x] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

Logs:

```OpenVPN settings:
|--User: [redacted]
|--Password: [redacted]
|--Verbosity level: 1
|--Run as root: no
|--Cyberghost settings:
 |--Network protocol: udp
 |--ClientKey: [redacted]
 |--Group: premium tcp europe
 |--Regions: united kingdom
System settings:
|--User ID: 1000
|--Group ID: 1000
|--Timezone:
|--IP Status filepath: /tmp/gluetun/ip
DNS over TLS settings:
 |--DNS over TLS provider:
  |--cloudflare
 |--Caching: enabled
 |--Block malicious: enabled
 |--Block surveillance: disabled
 |--Block ads: disabled
 |--Allowed hostnames:
  |--
 |--Private addresses:
  |--127.0.0.1/8
  |--10.0.0.0/8
  |--172.16.0.0/12
  |--192.168.0.0/16
  |--169.254.0.0/16
  |--::1/128
  |--fc00::/7
  |--fe80::/10
  |--::ffff:0:0/96
 |--Verbosity level: 1/5
 |--Verbosity details level: 0/4
 |--Validation log level: 0/2
 |--IPv6 resolution: disabled
 |--Update: every 24h0m0s
 |--Keep nameserver (disabled blocking): no
Firewall settings:
 |--VPN input ports:
 |--Input ports:
TinyProxy settings: disabled
ShadowSocks settings: disabled
HTTP Control server:
 |--Listening port: 8000
 |--Logging: true
Public IP check period: 12h0m0s
Version information: enabled
Updater: disabled

2020-10-28T18:40:36.781Z        INFO    storage: Merging by most recent 6400 hardcoded servers and 6400 servers read from /gluetun/servers.json
2020-10-28T18:40:37.012Z        INFO    routing: default route found: interface eth0, gateway 172.17.0.1
2020-10-28T18:40:37.014Z        INFO    routing: local subnet found: 172.17.0.0/16
2020-10-28T18:40:37.037Z        INFO    routing: default route found: interface eth0, gateway 172.17.0.1
2020-10-28T18:40:37.053Z        INFO    routing: adding route for 0.0.0.0/0
2020-10-28T18:40:37.055Z        INFO    openvpn configurator: checking for device /dev/net/tun
2020-10-28T18:40:37.056Z        INFO    firewall: enabling...
2020-10-28T18:40:37.264Z        INFO    firewall: enabled successfully
2020-10-28T18:40:37.265Z        INFO    Launching standard output merger
2020-10-28T18:40:37.265Z        INFO    dns over tls: falling back on plaintext DNS at address 1.1.1.1
2020-10-28T18:40:37.266Z        INFO    dns configurator: using DNS address 1.1.1.1 internally
2020-10-28T18:40:37.266Z        INFO    dns configurator: using DNS address 1.1.1.1 system wide
2020-10-28T18:40:37.268Z        INFO    http server: listening on 0.0.0.0:8000
2020-10-28T18:40:37.269Z        INFO    healthcheck: listening on 127.0.0.1:9999
2020-10-28T18:40:37.269Z        ERROR   openvpn: no server found for regions united kingdom and group "premium tcp europe"
2020-10-28T18:40:37.269Z        WARN    openvpn: loop exited
2020-10-28T18:40:37.269Z        WARN    healthcheck: context canceled: shutting down server
2020-10-28T18:40:37.269Z        WARN    http server: context canceled: exiting loop
2020-10-28T18:40:37.269Z        WARN    context canceled, shutting down
2020-10-28T18:40:37.269Z        INFO    Clearing ip status file /tmp/gluetun/ip
2020-10-28T18:40:37.769Z        WARN    healthcheck: server shut down
2020-10-28T18:40:37.769Z        WARN    http server: loop exited
2020-10-28T18:40:37.769Z        INFO    Shutdown successful
2020-10-28T18:40:37.770Z        INFO    routing: default route found: interface eth0, gateway 172.17.0.1
2020-10-28T18:40:37.770Z        INFO    routing: deleting route for 0.0.0.0/0

```

Configuration file:

```version: '3.7'
services:
  gluetun:
    container_name: gluetun
    image: qmcgaw/private-internet-access
    cap_add:
      - NET_ADMIN
    environment:
      - TZ=Europe/London 
    networks:
      - default
    restart: unless-stopped
    environment: 
      - VPNSP=cyberghost
      - USER=<redacted>
      - PASSWORD=<redacted>
      - CLIENT_KEY=<redacted>
      - REGION=United Kingdom
      - CYBERGHOST_GROUP=Premium TCP Europe
    volumes:
      - /dev/net:/dev/net
    ports:
      - 8000:8000/tcp

```

Host OS:

QTS 4.5.1

**Comment by adamsdavis1976** [comment]
Any suggestions for this issue please?

**Comment by qdm12** [comment]
Looking into it now

**Comment by qdm12** [comment]
It's fixed now, the issue was with the uppercase of the group name. If you want you can either

- repull the `latest` image once it's built
- change the group environment variable value to lowercase

Let me know if this doesn't solve it.

**Comment by adamsdavis1976** [comment]
@qdm12 many thanks this has resolved the issue

**Comment by qdm12** [comment]
Hello there, please note that the Cyberghost setup changed slightly from `:latest` and `:v3.8.0`, see the [newer readme](https://github.com/qdm12/gluetun/blob/1cc0f5fee9b9947bd71a6329dff5ea760e9aa862/README.md#vpn). Essentially you need to move both your client.key and client.crt in the /gluetun directory.
